blog/config/common/di/router.php

```diff
@@ -11,38 +11,35 @@
 use Yiisoft\Router\RouteCollectionInterface;
 use Yiisoft\Router\RouteCollectorInterface;
 use Yiisoft\Router\UrlGeneratorInterface;
-use Yiisoft\Yii\Debug\Viewer\Middleware\ToolbarMiddleware;
 
 /**
  * @var Config $config
  * @var array $params
+ * @var array $params['yiisoft/router-fastroute']
+ * @var bool $params['yiisoft/router-fastroute']['encodeRaw']
+ * @var array $defaultArguments 
+ * @psalm-suppress MixedArgument $routes
  */
 
 return [
     UrlGeneratorInterface::class => [
         'class' => UrlGenerator::class,
         'setEncodeRaw()' => [$params['yiisoft/router-fastroute']['encodeRaw']],
         'setDefaultArgument()' => ['_language', 'en'],
-        'reset' => function () {
-            $this->defaultArguments = ['_language', 'en'];
+        'reset' => function (array $defaultArguments = []) {
+            $defaultArguments = ['_language', 'en'];
         },
```

Comment:
Looks suspicious. What does it reset?

Reply:
Psalm level 1 testing no longer has an issue with $defaultArguments. I have removed array $defaultArguments = [] initialization and it works fine as before. I will remove it also from rossaddison/invoice. I cannot remember the exact error. 
![image](https://github.com/user-attachments/assets/23e552d2-7a26-4084-866b-195aee906440)